### PR TITLE
fix #12 - refactor ControlIn to return the number of bytes received

### LINKED ISSUE #12: ControlIn commands must return length of data received
ControlIn USB commands must return the number of bytes that were actually 
received. 

In typical USB applications its common to have bulk transfers where you don't 
know the length of the data that you're going to receive - so you set up a 
buffer with the maximum possible bytes and execute a ControlIn(..., buffer) 
command. After the command you check to see how many bytes were actually 
received.

Original issue reported on code.google.com by `navneet....@gmail.com` on 24 Feb 2015 at 12:19

### PR DESCRIPTION
**Pull request by adadurov**
As discussed in https://github.com/madwizard-thomas/winusbnet/issues/12, refactored ControlIn routines to return the number of bytes actually received.

### CODE REVIEW

File: WinUSBNet/API/WinUSBDevice.cs [hunk 1]
```diff
@@ -138,7 +138,7 @@ public string GetStringDescriptor(byte index, ushort languageID)
             return new string(chars);
         }
 
-        public void ControlTransfer(byte requestType, byte request, ushort value, ushort index, ushort length, byte[] data)
+        public uint ControlTransfer(byte requestType, byte request, ushort value, ushort index, ushort length, byte[] data)
```
Comment (madwizard-thomas) [inline WinUSBNet/API/WinUSBDevice.cs]: Maybe let this return int as well for consistency?

Comment (adadurov) [reply WinUSBNet/API/WinUSBDevice.cs]: Yeah, good idea. Will do.

Comment (adadurov) [reply WinUSBNet/API/WinUSBDevice.cs]: Done; please check the changes below.

**Comment by madwizard-thomas** [comment]
Thanks, merged and 2.0.0 released!

**Pull request merged**